doc/articles/faq.md

```diff
@@ -33,7 +33,9 @@ You can develop Uno Platform applications on Windows, macOS, or Linux. Supported
 
 Much like the new UI technologies from Microsoft, Uno Platform doesn’t support creation of new applications using VB.NET.
 
-If you have an existing VB.NET application which you would like to port/modernize for cross-platform scenarios with Uno Platform, you should be able to reuse all of your VB.NET business logic, packaged as NET standard libraries into a new Uno Platform app where only the new UI code would be defined in XAML with some glue in C#. Additionally, If you’d like to move any of your VB.NET code to C# you may be able to use automated tools such as https://converter.telerik.com
+If you have an existing VB.NET application which you would like to port/modernize for cross-platform scenarios with Uno Platform, you should be able to reuse all of your VB.NET business logic, packaged as NET standard libraries into a new Uno Platform app where only the new UI code would be defined in XAML with some glue in C#.
+To be exact, add "Class Library" VB project (not "Class Library (.Net Framework)", and not "Class Library (Universal Windows)"). When you want to use this library for UWP app compatible with phones (i.e., app that works on Windows Phones as Lumia), you have to change Target Framework to ".Net Standard 1.4" (in Project:Properties), as this is last .Net Standard that can be used with projects targetting Windows 10 15063. If you are targetting Windows 10 16299+, use .Net Standard 2.0. Then, in Uno heads for all platforms, add reference to this library (simplest way is to right click on "References" nodes inside these heads). You can use same Class Library also in your original, VB project. The same Class Library can also be used MAUI project, if you ever decide to port app from Uno to MAUI.
```

Comment:
I don't think it is necessary to mention MAUI as it just makes it more confusing (usually it is a FAQ itself - does Uno run on top of MAUI?). It gets even more confusing as we offered WebAssembly renderers for XF.

Reply:
As "an answer" to your "confusing ...  does Uno run on top of MAUI", I added "port app from Uno to MAUI". So, it is clear that Uno doesn't run on top of MAUI.
No-one knows what will last longer - MAUI, or Uno. Maybe MAUI will be abandoned by Microsoft, similarly to abandoning VB (I remember days when Microsoft claims "if you want to write programs for Windows, use VB, it is the best way for Windows programming"), or abandoning UWP ("best platform, unification of app code for all platforms"). But maybe Uno would stale, without new PRs. But any way, Class Library would work. it should work in any other future scenarios also.
So, mentioning MAUI is an argument: use Class Library, porting code to it would not be dead-end, your time invested in porting code to ClassLib would be sound investment in any future can be thought of.